WordPress/WordPressTest/Services/PostServiceWPComTests.swift

```diff
@@ -112,6 +112,27 @@ class PostServiceWPComTests: XCTestCase {
         expect(postFromDB.status).to(equal(.draft))
     }
 
+    func testTrashingAPostWillUpdateItsRevisionStatusAfterSyncProperty() {
+        // Arrange
+        let post = PostBuilder(context).with(statusAfterSync: .publish).withRemote().build()
+        let revision = post.createRevision()
+        try! context.save()
+
+        let remotePost = createRemotePost(.trash)
+        remoteMock.remotePostToReturnOnTrashPost = remotePost
+
+        // Act
+        waitUntil(timeout: 3) { done in
+         self.service.trashPost(post, success: {
+             done()
+         }, failure: self.impossibleFailureBlock)
+        }
+
+        // Assert
```

Comment:
Do you think it's worth it to include this too?

```swift
        expect(post.statusAfterSync).to(equal(.trash))
        expect(post.status).to(equal(.trash))
```

Reply:
Done in 43b2e686a4e50b14baf0230dc1de3d3b6108c736